server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinatorConfig.java

```diff
@@ -75,6 +75,10 @@ public String getLoadQueuePeonType()
     return "curator";
   }
 
+  @Config("druid.coordinator.curator.create.zknode.numThreads")
```

Comment:
`curator` prefix already implies `zookeeper`, I vote for:
`druid.coordinator.loadqueuepeon.curator.numThreads`

or if other pool sizes will be configurable too then:
`druid.coordinator.loadqueuepeon.curator.numCreateThreads`
`druid.coordinator.loadqueuepeon.curator.numCallbackThreads`
`druid.coordinator.loadqueuepeon.curator.numMonitorThreads`

Also, it would be more clear if threads were named accordingly to parameter names, ie:
`ZKNodeDeletionChecker-` -> `LoadQueuePeon-Monitor-`